lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -14,7 +14,7 @@ def self.included(base)
       base.option "--parent", "PARENT_NAME",  _("Name of parent hostgroup")
       base.option "--puppet-classes", "PUPPET_CLASS_NAMES", "",
         :format => HammerCLI::Options::Normalizers::List.new
-      base.option "--root-pass", "ROOT_PASSWORD",  _("Root password")
+      base.option "--root-password", "ROOT_PASSWORD",  _("Root password")
```

Comment:
This leads to duplicity in help:
```
   --root-pass ROOT_PASS                               Root password on provisioned hosts
   --root-password ROOT_PASSWORD                       Root password
```
The API defines the option `root-pass` while you are adding here another one.  The bug referenced from this fix talks just about host update/create so I'd suggest not to touch the hostgroup at all. I see the point in having consistency in host and hostgroup params  so I'd also suggest to create new issue for hostgroups to add `root-password` there with deprecation warning on `root-pass` (the same code you've just removed in hosts) and another issue to remove it two releases later.